package.json

```diff
@@ -110,7 +111,7 @@
     "accepts": "^1.3.8",
     "bytes": "^3.1.2",
     "cacheable-lookup": "^6.1.0",
-    "chalk": "^4.1.2",
+    "colorette": "^2.0.20",
```

Comment:
Why the change from chalk to colorette?

Reply:
For consistency. pino-pretty uses colorette, so most of the log coloring we do is actually already using colorette. It's also not ESM-only and we could actually consume any updates it gets.